dependencies.toml

```diff
@@ -74,34 +72,33 @@ jmh-gradle-plugin = "0.7.3"
 joor = "0.9.15"
 # Don't upgrade json-unit to 3.0.0 that requires Java 17
 json-unit = "2.38.0"
-jsoup = "1.18.3"
+jsoup = "1.21.1"
 junit4 = "4.13.2"
-junit5 = "5.12.0"
+junit5 = "5.13.4"
 # Don't upgrade junit-pioneer to 2.x.x that requires Java 11
 junit-pioneer = "1.9.1"
 jwt = "4.5.0"
 kafka = "3.9.0"
 kotlin = "2.1.10"
 kotlin-coroutine = "1.10.1"
 krotodc = "1.1.1"
-ktlint-gradle-plugin = "12.1.2"
+ktlint-gradle-plugin = "13.0.0"
 kubernetes-client = "7.3.1"
 logback12 = "1.2.13"
 logback13 = "1.3.14"
 logback14 = "1.4.14"
-logback15 = "1.5.16"
-micrometer = "1.14.4"
-micrometer-tracing = "1.4.3"
+logback15 = "1.5.18"
+micrometer = "1.15.2"
+micrometer-tracing = "1.5.2"
 micrometer-docs-generator = "1.0.2"
 # Don't uprade mockito to 5.x.x that requires Java 11
 mockito = "4.11.0"
 monix = "3.4.1"
-munit = "1.1.0"
-netty = "4.1.118.Final"
-netty-incubator-transport-native-io_uring = "0.0.26.Final"
+munit = "1.1.1"
+netty = "4.2.3.Final"
```

Comment:
I understood users using `netty 4.1.x` will need to upgrade to use the new Armeria version. I assume this will be noted in the release notes.